pkg/backend/httpstate/backend.go

```diff
@@ -1236,15 +1236,17 @@ func (b *cloudBackend) PromptAI(
 }
 
 func (b *cloudBackend) renderAndSummarizeOutput(
-	ctx context.Context, kind apitype.UpdateKind, stack backend.Stack, op backend.UpdateOperation, events []engine.Event,
+	ctx context.Context, kind apitype.UpdateKind, stack backend.Stack, op backend.UpdateOperation,
+	events []engine.Event, update client.UpdateIdentifier, updateMeta updateMetadata,
 ) {
+	dryRun := kind == apitype.PreviewUpdate
 	renderer := display.NewCaptureProgressEvents(
 		stack.Ref().Name(),
 		op.Proj.Name,
 		display.Options{
 			ShowResourceChanges: true,
 		},
-		true,
+		dryRun,
```

Comment:
Why was it `true` before? Which effect does this change have?

Reply:
There are differences in how preview vs. updates are rendered. e.g. `create` vs `creating` and some more expanded steps etc. This was a bug not handled properly in #19070